### ISSUE TITLE
Spades version detection bit wonky?

### ISSUE THREAD

**Issue by tseemann** [sanger-pathogens/circlator#56]
#It prints it as `.7.0` instead of `3.7.0` below?

I'm using `circlator  version 1.2.0`

```
Using bwa version 0.7.13 as /bio/linuxbrew/bin/bwa
Using nucmer version 3.1 as /bio/linuxbrew/bin/nucmer
Using prodigal version 2.6.2 as /bio/linuxbrew/bin/prodigal
Using samtools version 1.3 as /bio/linuxbrew/bin/samtools
Using spades version .7.0 as /bio/linuxbrew/bin/spades.py
```

PS. they only added a `--version` flag in 3.7.0

**Comment by martinghunt** [comment]
Thanks, definitely wonky! I'll fix it.

**Comment by martinghunt** [comment]
Fixed in release 1.2.1.

**Comment by tseemann** [comment]
Thank you. See you in September hopefully!
